galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -59,15 +59,15 @@ messages:
       - name: port
         type: int
   
-  - name: "IstioProxyVersionMismatch"
+  - name: "IstioProxyImageMismatch"
     code: IST0105
     level: Warning
-    description: "The version of the Istio proxy running on the pod does not match the version used by the istio injector."
-    template: "The version of the Istio proxy running on the pod does not match the version used by the istio injector (pod version: %s; injector version: %s). This often happens after upgrading the Istio control-plane and can be fixed by redeploying the pod."
+    description: "The image of the Istio proxy running on the pod does not match the image defined in the injection configuration."
+    template: "The image of the Istio proxy running on the pod does not match the image defined in the injection configuration (pod image: %s; injection configuration image: %s). This often happens after upgrading the Istio control-plane and can be fixed by redeploying the pod."
     args:
-      - name: proxyVersion
+      - name: proxyImage
         type: string
-      - name: injectionVersion
+      - name: injectionImage
```

Comment:
@zerobfd, this changes the description and template text but not the schema itself. Any compat. concerns here?

Reply:
Analysis message schema is not enforced and we are not currently generating schemas for the error and warning messages.

Reply:
LGTM.